src/sql/base/browser/ui/panel/panel.ts

```diff
@@ -24,6 +24,7 @@ export interface ITabbedPanelStyles {
 	activeBackgroundForVerticalLayout?: Color;
 	border?: Color;
 	activeTabContrastBorder?: Color;
+	contrastBorder?: Color;
```

Comment:
is this really needed? I don't see this being used, and I tested without this and it seems to be fine for all 3 themes.

Reply:
That was a leftover from testing, it wasn't supposed to be there.